app/src/test/java/com/cronus/zdone/TestTasksRepo.kt

```diff
@@ -18,7 +18,7 @@ class TestTasksRepo : TasksRepository {
     }
 
     override fun getTimeData(): Observable<TimeProgress> {
-        return Observable.just(TimeProgress(15, 85))
+        return Observable.just(TimeProgress(15, 85, 9999))
```

Comment:
is this the default? jfyi the default on server is `120`

Reply:
just a rando number for tests

Reply:
whoever updated that class to include the new field forgot to update the tests